### DISCUSSION TITLE
rust-course/关于 RefCell 的借用状态指示器的疑惑

### DISCUSSION THREAD

**Discussion by allandowney** [sunface/rust-course#463] [内容建议]
contents/advance/smart-pointer/cell-refcell.md:

> 121: 与 `Cell` 的 `zero cost` 不同，`RefCell` 其实是有一点运行期开销的，原因是它包含了一个字大小的“借用状态”指示器，该指示器在每次运行时借用时都会被修改，进而产生一点开销。
>

运行时借用时，应该如何理解？

**Comment by sunface** [comment]
这个运行时借用其实不是借用 borrowing 的一个子类 ，更多的就是对 `RefCell` 行为的描述 ，如果深究的话，这个描述确实不准确
